chrome/options.js

```diff
@@ -0,0 +1,18 @@
+function save_options() {
+  var autoSubmit = document.getElementById('auto-submit').checked;
+  localStorage.setItem('autoSubmit', autoSubmit);
```

Comment:
Maybe we should close settings on Save? I think a simple `window.close();` would do it.

Reply:
Correct, added `window.close();`